expertise/service/routes.py

```diff
@@ -29,6 +29,37 @@ def get_client():
         baseurl=flask.current_app.config['OPENREVIEW_BASEURL']
     )
 
+def format_error(status_code, description):
+    '''
+    Formulates an error that is in the same format as the OpenReview API errors
+
+    :param status_code: The status code determined by looking at the description
+    :type status_code: int
+
+    :param description: Useful information about the error
+    :type description: str
+
+    :returns template: A dictionary that zips all the information into a proper format
+    '''
+    # Parse status code
+    error_name = ''
+    if status_code == 400:
+        error_name = 'BadRequestError'
+    elif status_code == 403:
+        error_name = 'ForbiddenError'
+    elif status_code == 404:
+        error_name = 'NotFoundError'
+    elif status_code == 500:
+        error_name = 'InternalServerError'
+
+    template = {
+        'name': error_name,
+        'message': description,
+        'status': status_code
```

Comment:
Is having `status` necessary?

Reply:
I don't believe it is - I think it is redundant to have it here again